docs/src/index.md

```diff
@@ -37,9 +37,12 @@ programming.
   or the [Maximum likelihood estimation](@ref) problem using nonlinear
   programming.
 * Work through more in-depth tutorials at [JuMPTutorials.jl](https://github.com/jump-dev/JuMPTutorials.jl)
-* Join the [community forum](https://discourse.julialang.org/c/domain/opt/13) to
-  search for questions to commonly asked questions.
-  - Before asking a question, make sure to read [PSA: make it easier to help you](https://discourse.julialang.org/t/psa-make-it-easier-to-help-you/14757),
+
+!!! tip
+    Need help? Join the [community forum](https://discourse.julialang.org/c/domain/opt/13)
+    to search for questions to commonly asked questions.
+
+    Before asking a question, make sure to read [PSA: make it easier to help you](https://discourse.julialang.org/t/psa-make-it-easier-to-help-you/14757),
```

Comment:
PSA is not super straightforward for some people.
I would omit that from the the link and say something like:
Read the post: make it easier...